tracer/src/Datadog.Trace/Ci/Agent/CIWriterFileSender.cs

```diff
@@ -35,5 +36,35 @@ public Task SendPayloadAsync(EventsPayload payload)
 
             return Task.CompletedTask;
         }
+
+        public Task SendPayloadAsync(MultipartPayload payload)
+        {
+            var str = $"c:\\temp\\multipart-{Guid.NewGuid().ToString("n")}";
```

Comment:
I'm probably missing something (as this was there before), but should this really be Windows-specific? Couldn't/shouldn't we use `GetTempFileName`? Also, are those files ever cleaned up?